code/controllers/subsystem/statistics.dm

```diff
@@ -0,0 +1,7 @@
+//This isn't actually a subsystem. It exists solely to hold a game's round statistics, such as deaths.
+
+var/client/chara = null //The client who killed the most people
+var/client/frisk = null //The client who killed the least people
```

Comment:
I would not recommend putting the references in the var names themselves, as not everyone has played Undertale and will understand their purpose without having to look at the comment near the definition. It just makes it unreadable for some.